### ISSUE TITLE
Row and column position not announced automatically when navigating role="grid"

### ISSUE THREAD

**Issue by gdkraus** [FreedomScientific/VFO-standards-support#587]
### Summary

When navigating a ```role="grid"```, JAWS does not announce the row and column indexes as you navigate the grid. This is a different behavior than if you navigate a static ```<table>``` using the table navigation keys.

1. Go to the [ARIA Authoring Practices Data Grid example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/grid/dataGrids.html).
2. Turn the virtual cursor off.
3. Navigate the cells of the Data Grid.
4. Note that only the contents of the cell are announced, not the row and column index of the current cell.

However, if you press Control + Alt + NumPad 5, you can get the row and column index, even when interacting with a ```role="grid"```.

Is this the intended behavior of JAWS with ```role="grid"``` or is this a bug? NVDA announces the row and column index each time you move cells within the ```role="grid"```.

### Expected result

The contents of the cell are announced, and the row and column index information are announced too.

### Actual result

Only the cell contents are announced.

However, if you press Control + Alt + NumPad 5, you can get the row and column index, even when interacting with a ```role="grid"```.

### Example

* [role="grid"](https://www.w3.org/TR/wai-aria-practices-1.1/examples/grid/dataGrids.html)
* [Static table](https://www.w3.org/WAI/tutorials/tables/two-headers/)

### Additional Information

When navigating the data grid, JAWS seems to always announce when you enter column 1 when entering from column 0, but it does not make any other announcements for other columns.

#### JAWS version and build number

JAWS 2022.2110.36 ILM
 
#### Operating System and version

Windows 10

#### Browser and version:

Chrome 95.0.4638.69

**Comment by frex65** [comment]
Using JAWS 2022.2112.24, JAWS speaks the row and column numbers if the grid is traversed using Ctrl+Alt+arrow keys, but not using the arrow keys by themselves. Would this constitute expected behaviour?

**Comment by gdkraus** [comment]
@frex65, in this case using the unmodified arrow keys by themselves would be the expected interaction to use. Since it is a `role="grid"`, the UI component itself should handle keyboard focus as opposed to how you interact with a static `<table>` where you would use the virtual cursor's table navigation methods.

**Comment by gdkraus** [comment]
Checking to see if there is any progress on this. It appears to still be an issue with JAWS 2022.2207.25 and Chrome 103.0.5060.134.

**Comment by gdkraus** [comment]
Still checking on the status of this. This problem will impact many enterprise customers trying to use JAWS with interactive grids.
@stevefaulkner
